src/metabase/lib/util.cljc

```diff
@@ -1,11 +1,29 @@
 (ns metabase.lib.util
+  (:refer-clojure :exclude [format])
   (:require
    [clojure.set :as set]
+   [clojure.string :as str]
    [metabase.lib.options :as lib.options]
    [metabase.lib.schema :as lib.schema]
    [metabase.lib.schema.common :as lib.schema.common]
    [metabase.shared.util.i18n :as i18n]
-   [metabase.util.malli :as mu]))
+   [metabase.util.malli :as mu]
+   #?@(:clj
+       ([potemkin :as p]))
+   #?@(:cljs
+       ([goog.string :as gstring]
+        [goog.string.format :as gstring.format]))))
```

Comment:
I finally added `metabase.lib.util/format` so we can use it in both Clojure and ClojureScript